proto/process/agent.proto

```diff
@@ -1064,6 +1073,14 @@ message Manifest {
 	string version = 6;
 }
 
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2036
+message Namespace {
+	Metadata metadata = 1;
+	repeated string finalizers = 2;
+	string status = 3; // namespace status displayed by "kubectl get namespace"
```

Comment:
I see there is `NamespaceCondition` in `NamespaceStatus` , do we want to collect it ?
https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto#L2102

Reply:
I think we should, I will add it. It's not something I've seen before, but it looks like useful data for debugging:
https://www.redhat.com/sysadmin/troubleshooting-terminating-namespaces